python/fastscore/_schemer.py

```diff
@@ -0,0 +1,294 @@
+
+import random
+from itertools import groupby
+from datetime import datetime
+import re
+
+MAX_INT   = 2147483647
```

Comment:
not critical, just clarifying, bit a late but still 
* for get max int, max long of OS can be used some function?
* as I understand this functionallyty provided for Python 2 only, what about python 3? R is this code compatible between versions?